### ISSUE TITLE
Concurrent transactions - replacement transaction underpriced rejection

### ISSUE THREAD

**Issue by starspire** [ethereum/web3.js#1301]
Hi,
I would like to send a few transactions from one account in a concurrent way (not wait with next until the previous one would be mined). When leaving setting `nonce` to `web3js` I get `replacement transaction underpriced` error.  I can set `defaultBlock` as "pending" and then get nonce with `web3.eth.getTransactionCount` but until I explicitly set `nonce` I doesn't work either. Does `web3js` supports concurrent transactions like that? It's even more important with contract methods, but with `sendTransaction` I found a way to working example. I use version `1.0.0-beta27`. From the [docs](http://web3js.readthedocs.io/en/1.0/web3-eth-accounts.html?highlight=nonce) I read that when nonce is not set, it's get from `web3.eth.getTransactionCount()` - why then this example doesn't work with `web3.eth.defaultBlock = "pending"`?
```
import { accounts, privateKeys } from "./utils/constants";
const Web3 = require("web3");


describe("Concurrent transactions", () => {

	before(() => {
		this.web3 = new Web3("https://ropsten.infura.io");
		this.web3.eth.defaultBlock = "pending";
		this.web3.eth.accounts.wallet.add(privateKeys[0]);
		this.web3.eth.defaultAccount = accounts[0];
	});

	it("should send concurrent transactions", async () => {
		this.web3.eth.sendTransaction({
			from: accounts[0],
			to: accounts[1],
			value: this.web3.utils.toWei("0.01", "ether"),
			gas: 100000
		}).on("transactionHash", async (transactionHash) => {
			console.log(transactionHash);

			const nextNonce = await this.web3.eth.getTransactionCount(accounts[0], "pending");

			this.web3.eth.sendTransaction({
				from: accounts[0],
				to: accounts[2],
				value: this.web3.utils.toWei("0.02", "ether"),
				gas: 100000,
				// nonce: nextNonce
			}).on("transactionHash", (transactionHash) => console.log(transactionHash));
		});
	});
});
```

**Comment by jdkanani** [comment]
Try commenting `this.web3.eth.defaultBlock = "pending"` and un-commenting `nonce: nextNonce`

**Comment by starspire** [comment]
@jdkanani Of course this works in that way (that is the purpose of the snippet), but I'd like to know why it doesn't work with defaultBlock set.

**Comment by wupeaking** [comment]
https://ethereum.stackexchange.com/questions/39790/concurrency-patterns-for-account-nonce?newreg=d89f0aa0788b4da7a7fe844ff16903d0   @starspire

**Comment by nivida** [comment]
I'm closing this issue because increasing the ``nonce`` for locally signed transactions got implemented in 2.0.
